CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Added new command "dataSetLike(session, dataSetName, options)" to `Create` class, for use when creating a dataset with parameters like another data set
```

Comment:
Please change "new command" to "new method", to avoid confusion with CLI commands.

Reply:
Done ^^